Claroty/claroty-xdome/tests/test_phi_protocol.json

```
@@ -0,0 +1,72 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Claroty xDome",
+        "dialect_uuid": "9f47aa9f-52d7-4849-9462-cf7fc8bcd51a"
+      }
+    },
+    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over POCT1-A (PHI-containing protocol) was detected between 1.2.3.4 (Hematology Analyzer Gateway) and 5.6.7.8 (Glucose Meter)|5|device_asset_id=DZMKWPEVVL device_assignees=[] device_category=Industrial device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['00:50:56:94:b3:b8'] device_manufacturer=Roche device_model=CoaguChek Gateway device_network_list=['Corporate'] device_note=None device_os=Windows Server 2016 1607 device_site_name=Main device_subcategory=Clinical Lab device_type=Hematology Analyzer device_uid=f257cf93-017d-42b7-9292-75dc8a8e248f domain=None dst_ip=5.6.7.8 dst_mac=b8:78:79:13:5e:00 dst_port=36812 event_description=Communication over POCT1-A (PHI-containing protocol) was detected between 1.2.3.4 (Hematology Analyzer Gateway) and 5.6.7.8 (Glucose Meter) event_id=11111111 event_timestamp=2022-02-06T14:24:49.785636+00:00 event_type=Unencrypted PHI Protocol Communication geo_location=None msg_category=comm_event protocol=POCT1-A server_port=None src_ip=1.2.3.4 src_mac=00:50:56:94:b3:b8 src_port=3001"
+  },
+  "expected": {
+    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over POCT1-A (PHI-containing protocol) was detected between 1.2.3.4 (Hematology Analyzer Gateway) and 5.6.7.8 (Glucose Meter)|5|device_asset_id=DZMKWPEVVL device_assignees=[] device_category=Industrial device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['00:50:56:94:b3:b8'] device_manufacturer=Roche device_model=CoaguChek Gateway device_network_list=['Corporate'] device_note=None device_os=Windows Server 2016 1607 device_site_name=Main device_subcategory=Clinical Lab device_type=Hematology Analyzer device_uid=f257cf93-017d-42b7-9292-75dc8a8e248f domain=None dst_ip=5.6.7.8 dst_mac=b8:78:79:13:5e:00 dst_port=36812 event_description=Communication over POCT1-A (PHI-containing protocol) was detected between 1.2.3.4 (Hematology Analyzer Gateway) and 5.6.7.8 (Glucose Meter) event_id=11111111 event_timestamp=2022-02-06T14:24:49.785636+00:00 event_type=Unencrypted PHI Protocol Communication geo_location=None msg_category=comm_event protocol=POCT1-A server_port=None src_ip=1.2.3.4 src_mac=00:50:56:94:b3:b8 src_port=3001",
+    "event": {
+      "kind": "alert",
+      "category": ["authentication"],
+      "type": ["info"],
```

Can you change the parser to set the correct category and type for this event?
```suggestion
      "category": ["network"],
      "type": ["connection"],
```